src/domain/index.ts

```diff
@@ -28,6 +26,12 @@ export type {
   Token,
 } from "@hermeznetwork/hermezjs";
 
+export type HermezNetworkStatus = "online" | "offline";
```

Comment:
This should be `NetworkStatus` as it refers only to the wallet, the `HermezNetworkStatus` is just below this

Reply:
You're right! Good cath! Done.